azavea/cac-tripplanner
# Feature/kak/add event type#910

## Overview

Adds `Event` type to CMS models and admin interface.


### Demo

![cac_admin](https://user-images.githubusercontent.com/960264/33335131-2ca4c1b6-d43a-11e7-89ec-0bd30db8092d.png)

Admin interface has JS widgets for picking dates/times:
![cac_event_date_widget](https://user-images.githubusercontent.com/960264/33335133-2fb2df46-d43a-11e7-8332-ee1f6dc96594.png)

Event date form validation:
![date_validation](https://user-images.githubusercontent.com/960264/33336896-5d64ad66-d43f-11e7-9c15-dacc835df8dd.png)



### Notes

Also updated/removed admin JS references based on third-party library changes.


## Testing Instructions

 * Run migrations
 * Visit admin site: http://localhost:8024/admin
 * Should be able to add and edit events
 * Should be able to perform actions from list view (publish or unpublish in bulk)


## Checklist


Closes #910
